CHANGES

```diff
@@ -11,7 +11,7 @@ Version 8.11 (Unreleased)
 - Add configurable password validators to enforce password strength.
 - Send email to specific email when adding a new email rather than sending to all unverified email addresses.
 - Allow user to resend email verification to primary email address.
-
+- Fix bug with incorrect email address removal
```

Comment:
meh, I wouldn't add this. It's a pointless changelog entry, especially since it's just a bugfix for a previous one.